website/src/documentation/programming-guide.md

```diff
@@ -288,6 +310,9 @@ public static void main(String[] args) {
 {% github_sample /apache/beam/blob/master/sdks/python/apache_beam/examples/snippets/snippets.py tag:pipelines_constructing_reading
 %}
 ```
+```go
+lines := textio.Read(s, "protocol://path/file*.txt")
```

Comment:
Consider changing the path to be exactly the Java instance, or exactly the Python instance. I'd prefer the Java variant since it's file collocated at this time.
"protocol://path/to/some/inputData.txt"

Adding a third variant hurts cross language comprehension, but at least matching helps with folks who already know one language, but want to learn how to do the same thing another.

If we could get all the languages into tagged compillable snippets files like python does [1], that would be ideal, but that's a change for a separate PR.

[1]: https://github.com/apache/beam/blob/master/sdks/python/apache_beam/examples/snippets/snippets.py#L115